genai-perf/genai_perf/parser.py

```diff
@@ -634,7 +638,7 @@ def _add_endpoint_args(parser):
     endpoint_group.add_argument(
         "--backend",
         type=str,
-        choices=utils.get_enum_names(ic.OutputFormat)[2:],
+        choices=utils.get_enum_names(ic.OutputFormat)[0:2],
```

Comment:
We might need a better approach long-term as its own ticket, but I did clean-up here to fix the issue for now. See help prompt with the correct backends displayed below.
<img width="1247" alt="image" src="https://github.com/user-attachments/assets/adde4bdf-0909-4ac5-a10f-5869cfda0706">